packages/reakit/src/Box/__tests__/Box-test.tsx

```diff
@@ -1,5 +1,6 @@
 import * as React from "react";
 import { render } from "reakit-test-utils";
+import { verify } from "../../../../reakit-test-utils/src/verify";
```

Comment:
It should be something like this:
```suggestion
import { render, verify } from "reakit-test-utils";
```